Dockerfile

```diff
@@ -17,6 +17,7 @@ ADD primitives /creditcoin-node/primitives
 ADD runtime /creditcoin-node/runtime
 ADD sha3pow /creditcoin-node/sha3pow
 ADD chainspecs /creditcoin-node/chainspecs
+ADD test /creditcoin-node/test
```

Comment:
Do we really need this inside the docker image ?

Reply:
It will fail to compile without it.

>  => ERROR [builder 13/13] RUN source ~/.cargo/env && cargo build --release                                                                                                                                                                1.5s
------
 > [builder 13/13] RUN source ~/.cargo/env && cargo build --release:
#20 1.473 error: failed to load manifest for workspace member `/creditcoin-node/node`
#20 1.473 
#20 1.473 Caused by:
#20 1.473   failed to load manifest for dependency `creditcoin-node-rpc`
#20 1.473 
#20 1.473 Caused by:
#20 1.473   failed to load manifest for dependency `test-client`
#20 1.473 
#20 1.473 Caused by:
#20 1.473   failed to read `/creditcoin-node/test/client/Cargo.toml`
#20 1.473 
#20 1.473 Caused by:
#20 1.473   No such file or directory (os error 2)
------
executor failed running [/bin/bash -c source ~/.cargo/env && cargo build --release]: exit code: 101